Makefile

```diff
@@ -73,8 +73,20 @@ test-e2e: reinstall
 		-v \
 		-i $(ROOT_DIR)/os_migrate/localhost_inventory.yml \
 		-e os_migrate_data_dir=$(ROOT_DIR)/tests/func/tmpdata \
+		-e os_migrate_src_osm_server_flavor=m1.xtiny \
+		-e os_migrate_src_osm_server_image=cirros-0.4.0-x86_64-disk.img \
+		-e os_migrate_src_router_external_network=public \
+		-e os_migrate_dst_router_external_network=public \
 		-e os_migrate_src_validate_certs=False \
 		-e os_migrate_dst_validate_certs=False \
+		-e os_migrate_src_conversion_host_name=osm_uch_src \
+		-e os_migrate_dst_conversion_host_name=osm_uch_dst \
+		-e os_migrate_src_conversion_host_flavor=m1.medium \
+		-e os_migrate_dst_conversion_host_flavor=m1.medium \
+		-e os_migrate_src_client_key=~/.ssh/id_rsa.pub \
+		-e os_migrate_dst_client_key=~/.ssh/id_rsa.pub \
+		-e os_migrate_conversion_host_key=~/.ssh/id_rsa \
+		-e os_migrate_conversion_host_image=v2v-conversion-host-appliance-devel.qc2 \
```

Comment:
Eventually we may want to move to passing a file here so that these params are generated by the CI environment and not hardcoded in the Makefile. Let's do it in a subsequent patch though, no need to churn on this one.